spec/spec_helper.rb

```diff
@@ -35,8 +35,41 @@ def inside_of_folder(folder)
   end
 end
 
+# See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
 RSpec.configure do |config|
-  config.expect_with(:rspec) { |c| c.syntax = :should }
   config.include SpecHelpers::InstanceMethods
   config.extend SpecHelpers::ClassMethods
+
+  config.filter_run_when_matching :focus
```

Comment:
run with line numbers ? ... also if you like that can this not got into `~/.rspec` ?